HHS/OPRE-OPS
# OPS-1156 View Agreement Details (part 1)

## What changed

* a new page to view an Agreement with the first tab to view Agreement Details implemented
  * top summary sections includes cards for Total Budget Lines and Total Agreement Value
  * details section includes Agreement metadata and Notes (with mock data) and History (with mock data) 
* NOTE: the second tab for Budget Lines is not implement in this PR
* new links from the Agreement list (on the Agreement name) to the new Agreement details page

## Issue

- #1156
  - #1210
  - #1211 
  - #1239 
  - #1240 
  - #1241

## How to test

* view the Agreements list page
* follow links from the Agreement name to the detail page
* verify the details page
* create an Agreement with budget lines with date_needed in the next three year
* verify the details page includes the bar chart with those

## Screenshots

### Agreements list page with links to view details
![image](https://github.com/HHS/OPRE-OPS/assets/127965976/215728b9-8046-4fbb-ae4d-7ce2a5e74163)

### Agreement details page
**_NOTE: This image shows bar chart with FY 2043, but only the next three fiscal years will be shown_**
![image](https://github.com/HHS/OPRE-OPS/assets/127965976/57f011b5-cc35-42e7-8375-c8377a939fa6)
